### PR TITLE
misc post-crd fixes

### PR DESCRIPTION
**Pull request by detiber**
- clusterawsadm: consolidate controller roles/profiles, since there is only a single controller manager now
- clusterawsadm: fixup glog error
- aws provider controller manager: fixup argument error for the aws provider controller manager statefulset.

### CODE REVIEW

File: pkg/cloud/aws/services/cloudformation/bootstrap.go [hunk 1]
```diff
@@ -27,34 +29,21 @@ import (
...
-			cloudformation.Ref("AWSIAMRoleClusterController"),
-			cloudformation.Ref("AWSIAMRoleControlPlane"),
-		},
-	}
-
-	template.Resources["AWSIAMManagedPolicyMachineController"] = cloudformation.AWSIAMManagedPolicy{
-		ManagedPolicyName: iam.NewManagedName("machine-controller"),
-		Description:       `For the Kubernetes Cluster API Provider AWS Machine Controller`,
-		PolicyDocument:    machineControllerPolicy(accountID),
+	template.Resources["AWSIAMManagedPolicyControllers"] = cloudformation.AWSIAMManagedPolicy{
```
Comment (detiber) [inline pkg/cloud/aws/services/cloudformation/bootstrap.go]: @randomvariable considering the post-crd deployment model of using a shared controller manager, it seems to make sense to collapse these roles/profiles down. Thoughts?

Comment (randomvariable) [reply pkg/cloud/aws/services/cloudformation/bootstrap.go]: For now, maybe. But we might want to separate them back out at some point.

The GCP provider actually takes a flag to say which actuator your starting.

I was thinking of the following use case:

You have a central management cluster that's responsible for spinning up clusters and control plane nodes, and that's not pivoted.

The target cluster may still have a machine actuator which deals with machinesets or machines that the cluster autoscaler is creating / modifying.

Comment (randomvariable) [reply pkg/cloud/aws/services/cloudformation/bootstrap.go]: Happy to lgtm it with whatever you think.

**Comment by detiber** [comment]
@vincepri PRs are squashed on merge automatically: https://github.com/kubernetes/test-infra/blob/2005cfe3b3f0148cfcfbb7272a9dee64e4a87211/prow/config.yaml#L418

/hold cancel

**Comment by vincepri** [comment]
Nice, thanks for the link :)

**Pull request merged**